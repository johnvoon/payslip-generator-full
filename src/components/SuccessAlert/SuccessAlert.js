import React from "react";
import PropTypes from "prop-types";

const SuccessAlert = ({ message }) => {
  return (
    <div className="alert alert-success" role="alert">
      <span className="" aria-hidden="true" />
      <span className="sr-only">Success:</span>
      {message}
    </div>
  );
};

SuccessAlert.propTypes = {
  message: PropTypes.string.isRequired
};

export default SuccessAlert;
