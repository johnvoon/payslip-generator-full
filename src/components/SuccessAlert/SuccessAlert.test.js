import React from "react";
import SuccessAlert from "./SuccessAlert";
import { shallow } from "enzyme";

describe("SuccessAlert", () => {
  const renderedComponent = shallow(<SuccessAlert message="Success" />);

  it("should match snapshot", () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
