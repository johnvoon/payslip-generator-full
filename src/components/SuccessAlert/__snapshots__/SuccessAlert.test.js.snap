exports[`SuccessAlert should match snapshot 1`] = `
<div
  className="alert alert-success"
  role="alert">
  <span
    aria-hidden="true"
    className="" />
  <span
    className="sr-only">
    Success:
  </span>
  Success
</div>
`;
